middleware/cache/README.md

```diff
@@ -32,8 +33,9 @@ cache [TTL] [ZONES...] {
   number of packets we cache before we start evicting (LRU). **TTL** overrides the cache maximum TTL.
 * `denial`, override the settings for caching denial of existence responses, **CAPACITY** indicates the maximum
   number of packets we cache before we start evicting (LRU). **TTL** overrides the cache maximum TTL.
-
-There is a third category (`error`) but those responses are never cached.
+  There is a third category (`error`) but those responses are never cached.
+* `prefetch`, will prefetch popular items when there are about to be expunged from the cache.
+  Popular means **AMOUNT** queries seen within **DURATION**, which defaults to 1m.
```

Comment:
Popular means **AMOUNT** queries have been seen no gaps of **DURATION** or more between them. **DURATION** defaults to 1m.